packages/admin/admin/src/form/ChipSelect.tsx

```diff
@@ -0,0 +1,163 @@
+            menuItem: typeof MenuItem;
+            inputBase: typeof MuiInputBase;
+        }>,
+        Omit<ChipProps, "children" | "onChange"> {
+    children?: React.ReactNode;
+    getOptionLabel?: (option: T) => string;
+    getOptionValue?: (option: T) => string;
+    options?: T[];
+    selectedOption?: T;
+    onChange: (event: SelectChangeEvent) => void;
+    fullWidth?: boolean;
+}
+
+const ChipInput = ({
+    chipProps,
+    inputBaseProps,
+    inputRootProps,
+    ...restProps
+}: InputBaseProps & {
+    chipProps?: Omit<ChipProps, "children">;
+    inputBaseProps?: InputBaseProps;
+    inputRootProps?: React.ComponentPropsWithoutRef<"div">;
+}) => {
+    const fallBackLabel = <FormattedMessage id="comet.admin.form.chipselect.placeholder" defaultMessage="Select a value" />;
+
+    return (
+        <ChipInputRoot {...inputRootProps}>
+            <Chip
+                icon={<ChevronDown />}
+                label={restProps.value?.toString().length ? restProps.value : fallBackLabel}
+                variant="filled"
+                {...chipProps}
+            />
+            <InputBase
+                sx={{
+                    position: "absolute",
+                    top: 0,
+                    left: 0,
+                    right: 0,
+                    bottom: 0,
+                    opacity: 0,
+                }}
+                {...inputBaseProps}
+                {...restProps}
+            />
+        </ChipInputRoot>
+    );
+};
+
+export function ChipSelect<T = string>(inProps: ChipSelectProps<T>) {
+    const {
+        options,
+        getOptionLabel = (option: T) => {
+            if (typeof option === "object" && option !== null) {
+                if ((option as any).label) return String((option as any).label);
+                if ((option as any).id) return String((option as any).id);
+                if ((option as any).value) return String((option as any).value);
+                return JSON.stringify(option);
+            } else {
+                return String(option);
+            }
+        },
+        getOptionValue = (option: T) => {
+            if (typeof option === "object" && option !== null) {
+                if ((option as any).value) return String((option as any).value);
+                if ((option as any).id) return String((option as any).id);
+                return JSON.stringify(option);
+            } else {
+                return String(option);
+            }
+        },
+        selectedOption,
+        children,
+        slotProps,
+        onChange,
+        fullWidth,
+        className,
+        ...chipProps
+    } = useThemeProps({
+        props: inProps,
+        name: "CometAdminChipSelect",
+    });
+
+    return (
+        <Root {...slotProps?.root} sx={fullWidth ? { ...slotProps?.root?.sx } : { ...slotProps?.root?.sx, width: "fit-content" }}>
+            <Select
+                {...slotProps?.select}
+                value={selectedOption || ""}
```

Comment:
When using `Select` and `ChipSelect` in the exact same way, the `Select` shows the options label, while the `ChipSelect` show's the value. This should be consistent between the components. 

<img width="271" alt="image" src="https://github.com/vivid-planet/comet/assets/6264317/d06c8d4c-df17-44ce-881d-ccbaaafa6131">

```tsx
const options = [
    { value: "option-value", label: "Option Label" },
];
// ...
<SelectField name="select" label="Select" fullWidth>
    {options.map((option) => (
        <MenuItem key={option.value} value={option.value}>
            {option.label}
        </MenuItem>
    ))}
</SelectField>
<ChipSelectField name="select" label="Chip Select" fullWidth>
    {options.map((option) => (
        <MenuItem key={option.value} value={option.value}>
            {option.label}
        </MenuItem>
    ))}
</ChipSelectField>
```

Reply:
https://github.com/vivid-planet/comet/pull/2186/commits/64067616a48fd9a3b6a88441005707739d1dec0d